### PR TITLE
Add steps: image output and registration

### PR DESCRIPTION
**Pull request by boaz0**

### CODE REVIEW

File: src/Routes/ImageManager/steps/packages.js [hunk 1]
```diff
@@ -3,19 +3,14 @@ import componentTypes from '@data-driven-forms/react-form-renderer/component-typ
...
     {
       component: componentTypes.PLAIN_TEXT,
       name: 'google-cloud-text-component',
-      label: (
-        <Text>
-          Add optional additional packages to your image by searching available
-          packages.
-        </Text>
-      ),
+      label: <Text>Add packages to your RHEL 8.3 image.</Text>,
```
Comment (karelhala) [inline src/Routes/ImageManager/steps/packages.js:13]: We might want to have dynamic label here, based on what was selected in image output step.

File: src/components/form/ReviewStep.js [hunk 2]
```diff
@@ -14,53 +14,64 @@ const ReviewStep = () => {
...
-      <Text component={TextVariants.h3}>Image output</Text>
+      <Text component={TextVariants.h1}>Image output</Text>
       <TextList
         component={TextListVariants.dl}
         data-testid="review-image-output"
       >
         <TextListItem component={TextListItemVariants.dt}>Release</TextListItem>
         <TextListItem component={TextListItemVariants.dd}>
-          Add value
+          Red Hat Enterprise Linux (RHEL) 8.3
```
Comment (karelhala) [inline src/components/form/ReviewStep.js:28]: Same as above, this should be the value selected in first step

File: src/Routes/ImageManager/steps/packages.js (hunk 1 above)
Comment (boaz0) [reply src/Routes/ImageManager/steps/packages.js]: DONE

File: src/components/form/ReviewStep.js (hunk 2 above)
Comment (boaz0) [reply src/components/form/ReviewStep.js:28]: DONE

**Comment by biancarosa** [comment]
@boaz0 on this edge-api PR i added some examples of requests
https://github.com/RedHatInsights/edge-api/pull/38

and errors are coming like that:
`{"Code":"ERROR","Status":500,"Title":"Something wrong happened."}`

i'm happy to change and we can talk about contract for features that are not ready on img builder yet (registration, iso output, etc)

**Review by karelhala — APPROVED** [review]

**Pull request merged**